### PR TITLE
[Manual Mapping] Maintain the mapping for more than 1 URL in the localstorage.

### PR DESCRIPTION
**Pull request by atopper**
## Description
The mapping for 1 URL is deleted if the URL is changed, and a new mapping is started.  Work can be lost easily this way.  This change maintains multiple URls in the local storage

Eventually, perhaps, we should set a TTL of URL mappings.  They could grow over time.

## Motivation and Context

Ensure mappings are saved.

## How Has This Been Tested?

Manually and ran tests.

## Screenshots (if appropriate):
image

## Types of changes
I'll say bug, although 'new feature' might apply.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: js/dist/helix-importer.js [hunk 1]
```diff
@@ -75284,7 +75284,7 @@ const serializer = (strict, json, $, _) => {
       case _types_js__WEBPACK_IMPORTED_MODULE_0__.ARRAY: {
         if (type)
           return as([type, [...value]], value);
-
```
Comment (catalan-adobe) [inline js/dist/helix-importer.js:75287]: `js/dist/*` files should not be modified directly. They are external dependencies built by the project.

File: js/sections-mapping/sections-mapping.import.js [hunk 2]
```diff
@@ -15,19 +16,7 @@ export function generateDocumentPath({ document, url }) {
...
-    // TODO - support multiple mappings
-    // const found = mData.find((m) => m.url === url);
-    // if (found) {
-    //   return found;
-    // }
-  }
-
-  return null;
+  return getImporterSectionMapping(url) ?? null;
 }
```
Comment (catalan-adobe) [inline js/sections-mapping/sections-mapping.import.js:20]: Let's use `getImporterSectionMapping` directly and get rid of `getSectionsMappingData`

File: js/sections-mapping/sections-mapping.import.js [hunk 3]
```diff
@@ -90,7 +79,7 @@ export default {
         }
       }
     });
-    
+
```
Comment (catalan-adobe) [inline js/sections-mapping/sections-mapping.import.js:82]: Let's remove this empty line.

**Review by catalan-adobe — COMMENTED** [review]
Thanks for the contribution! Overall LGTM. Few cosmetic comments below :)

**Comment by atopper** [comment]
Do you think instead of the URL as the 'key', to use the hostname instead, so that any subpages are saved in the same mapping?
Probably not needed for the pre-sales demo, but it is something that I keep wondering about.

File: .gitignore [hunk 4]
```diff
@@ -2,4 +2,6 @@
 coverage/*
 logs/*
 node_modules/*
+# js/dist/* (maybe?)
```
Comment (atopper) [inline .gitignore:5]: @catalan-adobe  Is this a good idea (commented for now)? ☝️

File: js/dist/helix-importer.js (hunk 1 above)
Comment (atopper) [reply js/dist/helix-importer.js:75287]: My Intellij keeps formatting it for me 🙄.  I'll get there.  Going to command line now.

File: .gitignore (hunk 4 above)
Comment (catalan-adobe) [inline .gitignore:5]: No. We might debate on the naming but historically this folder contain dependency "vendor" files we need in the project.

File: js/shared/mappings.js
Comment (catalan-adobe) [inline js/shared/mappings.js:1]: Neatpicking but everything about sections mapping is very specific so for now I would prefer to see this under `/js/sections-mapping/utils.js` (or `utils.ui.js`).

**Review by catalan-adobe — CHANGES_REQUESTED** [review]
Thanks for the changes! I would just like `/js/shared/mapping.js` to be moved to `/js/sections-mapping/utils.js` (or `utils.ui.js`)

File: js/shared/mappings.js
Comment (atopper) [reply js/shared/mappings.js:1]: Being picky is appreciated!

**Pull request closed without merging**